### PR TITLE
feat(ef-cli): show proper message if not able to drop DB Closes #5317

### LINKED ISSUE #5317: dotnet ef database drop always ask if it wants to drop the database regardless of the existence of the db
The command always prompts for `Are you sure you want to drop the database ...` without verifying if it exists. Running it twice makes it appear as if it hasn't done any work the first time around.

### PR DESCRIPTION
**Pull request by laskoviymishka**
**Please check if the PR fulfills these requirements**

- [x] The code builds and tests pass (verified by our automated build checks)
- [x] Commit messages follow this format
```
    Summary of the changes
    - Detail 1
    - Detail 2

    Fixes #bugnumber
```
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Code meets the expectations our engineering guidelines. https://github.com/aspnet/Home/wiki/Engineering-guidelines.

Please review the guidelines for CONTRIBUTING.md for more details.

### CODE REVIEW

**Comment by smitpatel** [comment]
Is reason for `context.Database.EnsureDeleted()` only because database cannot exists? What about network errors?

File: src/Microsoft.EntityFrameworkCore.Tools.Core/Properties/ToolsCoreStrings.resx [hunk 1]
```diff
@@ -249,4 +249,7 @@ Change your target project to the migrations project by using the Package Manage
   <data name="NoParameterlessConstructor" xml:space="preserve">
     <value>No parameterless constructor was found on '{contextType}'. Either add a parameterless constructor to '{contextType}' or add an implementation of 'IDbContextFactory&lt;{contextType}&gt;' in the same assembly as '{contextType}'.</value>
   </data>
+  <data name="LogNotExistDatabase" xml:space="preserve">
+    <value>Cannot drop '{name}'. Are you sure it exists?</value>
```
Comment (rowanmiller) [inline src/Microsoft.EntityFrameworkCore.Tools.Core/Properties/ToolsCoreStrings.resx]: I think this wording is better. `EnsureDeleted` (and this command) are designed to be a "delete it if it is there" operation, so the database not being there is still a valid scenario.

> Database did not exist, no action was taken.

**Comment by rowanmiller** [comment]
@smitpatel you should get an exception if there is any issue deleting the database. The return value on the method is designed to tell you if there was a database to delete or not.

File: src/Microsoft.EntityFrameworkCore.Tools.Core/Properties/ToolsCoreStrings.resx (hunk 1 above)
Comment (laskoviymishka) [inline src/Microsoft.EntityFrameworkCore.Tools.Core/Properties/ToolsCoreStrings.resx]: @rowanmiller fixed

**Comment by rowanmiller** [comment]
This is :shipit: from my perspective.

**Comment by rowanmiller** [comment]
@laskoviymishka thanks for the contribution!

**Comment by laskoviymishka** [comment]
@rowanmiller glad to help!

**Pull request merged**